advanced/Plot data extraction demo.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plot data extraction demo\n",
+    "\n",
+    "This notebook demonstrate how to extract point data from plot using plot data extraction endpoint.\n",
+    "\n",
+    "Current version only support curve plots similar to the example which has no legend."
```

Comment:
The currrent version only supports line plots without a legend, similar to the example.